connectors/sources/abs.py

```diff
@@ -94,7 +94,7 @@ def get_default_configuration(cls):
             },
             "retry_count": {
                 "value": DEFAULT_RETRY_COUNT,
-                "label": "How many retry count for fetching rows on each call",
+                "label": "How many retry count for failed requests",
```

Comment:
I would make it less dev-ish

`How many retries when calling the Azure server`